src/lwc/edaSetupHealthCheck/edaSetupHealthCheck.css

```diff
@@ -0,0 +1,4 @@
+.parentComponent {
+    width: auto;
+    background-color: transparent;
```

Comment:
Let's avoid any custom styling we can. I believe we can get away with this. A div should default to transparent background color.